### ISSUE TITLE
Controller is loaded twice

### ISSUE THREAD

**Issue by ajoah** [roots/sage#2126]
## Submit a feature request or bug report

- [X] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/guidelines/blob/master/CONTRIBUTING.md)
- [ ] This is a feature request
- [X] This is a bug report
- [X] This request isn't a duplicate of an [existing issue](https://github.com/roots/sage/issues)
- [X] I've read the [docs](https://roots.io/sage/docs) and [NPM Debugging Guidelines post](https://discourse.roots.io/t/npm-debugging-guidelines-failed-npm-install-bower-install-or-gulp-build-read-this/3060) and followed them (if applicable)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/c/sage) forums

---
I installed master branch.
I modified FrontPage controller to :
```
<?php
namespace App\Controllers;

use Sober\Controller\Controller;

class FrontPage extends Controller
{
	public function __construct(){
		echo 'test';
	}
}
```
(This code is only to see the problem, it is not "production code")
**What is the current behavior?**

On the frontpage, I see "testtest", Which means that FrontPage is instantiate twice.

**What is the expected or desired behavior?**

On the frontpage, I should see "test". Which means that FrontPage is instantiate only once.

---

## Bug report

**Please provide steps to reproduce, including full log output:**
Just install sage with command ` composer create-project roots/sage sage-master dev-master`
Switch to this theme.
Maybe you should set a static page as homepage in WP settings.
Go to website url root.

I looked backtrace and a first loaded is due to soberwp/Controller filters : https://github.com/soberwp/controller/blob/master/controller.php#L107
and a second load is due to this apply_filters : https://github.com/roots/sage/blob/master/app/filters.php#L51

**Comment by darrenjacoby** [comment]
You shouldn't echo from `__construct `

If you're trying to return a value, use a public function and return it rather than echo.

**Comment by ajoah** [comment]
Yes, of course. It is only an example to easily see and reproduce the problem : FrontPage object is instanciate twice. It is a performance issue.

**Comment by darrenjacoby** [comment]
@ajoah can you create an issue on https://github.com/soberwp/controller and I will take a look at it.
